### PR TITLE
[WIP] add makefu nixpkgs

### PR DESCRIPTION
**Pull request by makefu**
the repository is shared between multiple parties.
I use my sub-directory with the nur.nix file to export my packages

The following points apply when adding a new repository to repos.json

- [x] I ran nur/format-manifest.py after updating `repos.json` (We will use the same script in travis ci to make sure we keep the format consistent)
- [x] By including this repository in NUR I give permission to license the
content under the MIT license.

### CODE REVIEW

**Comment by Mic92** [comment]
getting status of '/nix/store/p4ql2xs5vqxvdc4fyywn1wirbkklraxi-stockholm/makefu/5pkgs/nur.nix': No such file or directory

**Comment by makefu** [comment]
i tried to re-run the ci but it seems to be "in the queue" ...

**Comment by Mic92** [comment]
Now the error is different: 

```
file 'stockholm/lib' was not found in the Nix search path (add it using $NIX_PATH or -I), at /nix/store/f353x6n4bjpsz5ykjfqh5iyk99lbpas0-stockholm/makefu/5pkgs/default.nix:1:13
```

**Comment by makefu** [comment]
yep, working on it (need to clean up my own package set :D)

File: repos.json [hunk 1]
```diff
@@ -25,6 +25,10 @@
         "ma27": {
             "url": "https://gitlab.com/Ma27/nur-packages"
         },
+        "makefu": {
+            "file": "makefu/5pkgs/nur.nix",
+            "url": "http://cgit.euer.krebsco.de/stockholm"
```
Comment (Mic92) [inline repos.json]: Fetching from your gitlab instance would be a bit faster because we can check gitlab's rss feed.

Comment (Mic92) [reply repos.json]: Also it would not be too hard to add cgit support also: https://github.com/nix-community/NUR/blob/master/nur/prefetch.py#L40

Comment (makefu) [reply repos.json]: I can also push to a gitlab instance.

btw test says:
```
access to URI 'https://github.com/Pistos/diakonos' is forbidden in restricted mode
Failed to updated repository manveru
```
hope this is fine

File: repos.json [hunk 2]
```diff
@@ -25,6 +25,10 @@
         "ma27": {
             "url": "https://gitlab.com/Ma27/nur-packages"
         },
+        "makefu": {
+            "file": "makefu/5pkgs/nur.nix",
+            "url": "https://git.shackspace.de/krebs/stockholm"
```
Comment (Mic92) [inline repos.json]: Now with https!

**Comment by Mic92** [comment]
If others members of the repository also want to share their packages, please do it through the same repository.

**Pull request merged**